conf/waivers

```diff
@@ -0,0 +1,106 @@
+# which are not available in their final form in the Anaconda environment
+# - see https://github.com/ComplianceAsCode/content/issues/9746
+/hardening/anaconda(/with-gui)?/[^/]+/firewalld_sshd_port_enabled
+    rhel >= 8
+
+# https://github.com/OpenSCAP/openscap/issues/1880
+# needs to be remediated more than once due to rule ordering issues
+/hardening/oscap(/with-gui)?/[^/]+/configure_bashrc_exec_tmux
+/hardening/oscap(/with-gui)?/[^/]+/no_tmux_in_shells
+/hardening/oscap(/with-gui)?/[^/]+/configure_usbguard_auditbackend
+    rhel >= 8
+/hardening/oscap(/with-gui)?/[^/]+/configure_bashrc_tmux
+/hardening/oscap(/with-gui)?/stig(_gui)?/postfix_prevent_unrestricted_relay
+    rhel == 8
+
+# rule ordering issue - accounts_password_pam_retry is checked first and passes,
+# and a later enable_authselect remediation breaks it
+# - see https://github.com/OpenSCAP/openscap/issues/1880
+/hardening/.+/accounts_password_pam_retry
+    rhel >= 8
+
+# https://github.com/ComplianceAsCode/content/pull/10499
+# these are missing the required partitions, making the oscap Anaconda addon
+# error out during remediaton
+/hardening/anaconda/cis_(server|workstation)_l1
+    ssg < '0.1.68' and status == 'error' and \
+        note.startswith('RuntimeError: installation failed')
+
+# bz1828871, won't be fixed on rhel7
+# needs to be remediated more than once due to rule ordering issues
+/hardening/anaconda(/with-gui)?/[^/]+/postfix_network_listening_disabled
+    rhel == 7
+
+# bz1778661 abrt package
+# - seems to be less predictable on new upstream content
+/hardening/anaconda(/with-gui)?/e8/rpm_verify_(ownership|permissions)
+    Match(rhel == 7, sometimes=True)
+
+# bz1825810 or maybe bz1929805
+# can be reproduced mostly reliably (95%) both via anaconda and oscap CLI,
+# but apparently we don't really care about old releases
+/hardening/[^/]+/with-gui/stig_gui/sysctl_net_ipv4_ip_forward
+    Match(rhel == 7, sometimes=True)
+/hardening/[^/]+/with-gui/cis_workstation_[^/]+/sysctl_net_ipv4_ip_forward
+    rhel == 8
+
+# https://github.com/ComplianceAsCode/content/issues/10424
+/hardening/anaconda(/with-gui)?/[^/]+/service_nftables_disabled
+    ssg >= '0.1.68'
+
+# caused by us not caring whether the VM has it installed,
+# - remediation should fix it, but doesn't -- possibly caused by
+#   https://github.com/RHSecurityCompliance/contest/issues/15
+/hardening/oscap(/with-gui)?/[^/]+/package_scap-security-guide_installed
+    Match(True, sometimes=True)
+
+# TODO: something new? .. RHEL-8 on e8 and ism_o, RHEL-7 e8
+#  - seems to not happen on latest 8.9 nightlies ??
+#  - on latest 7.9, but upstream 2023/05 content
+/hardening/oscap/[^/]+/package_rear_installed
+    Match(rhel <= 8, sometimes=True)
+
+# happens normally only on rhel-7 e8, but it's more widespread with GUI,
+# caused by the 'gdm' package
+/hardening/[^/]+/with-gui/[^/]+/rpm_verify_(ownership|permissions)
+    rhel == 9
+/hardening/oscap/with-gui/e8/rpm_verify_(ownership|permissions)
+    rhel == 7
```

Comment:
I'll check it. It should have been fixed (at least for RHEL9) according to package version - https://bugzilla.redhat.com/show_bug.cgi?id=1878119

Reply:
Rhel8 fix broke rhel9 - https://bugzilla.redhat.com/show_bug.cgi?id=2209266

I wasn't able to reproduce it on RHEL7. In case you manage to reproduce it, you can clone that BZ.

Reply:
It's actually a different bug about misconfigured `/run` and `tmpfiles.d`, which was fixed on RHEL-9 (and maybe 8 too), https://bugzilla.redhat.com/show_bug.cgi?id=2209294 .

I'll add all 3 to the waive comment.